### PR TITLE
Tests cases for signin API

### PR DESCRIPTION
**Pull request by ashutoshsingh0223**
- Success test case
- Wrong password failure
- Wrong request body failure

### CODE REVIEW

File: embark/users/test_users.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        pass
+
+    def test_signin(self):
+        """
+        Right password would redirect to home
+        Returns:
+
+        """
+        response = self.client.post('/signin', {'email': 'testuser', 'password': '12345'})
+        self.assertEqual(response.status_code, 302)
```
Comment (MaximilianWagner) [inline embark/users/test_users.py:23]: You could use the constant http.HTTPStatus.FOUND for readability from https://docs.python.org/3/library/http.html#module-http.

File: embark/users/test_users.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+    def test_signin_wrong_password(self):
+        """
+        Wrong password would render the same page again with 200 status code
+        Returns:
+
+        """
+        response = self.client.post('/signin', {'email': 'testuser', 'password': '1234'})
+        self.assertEqual(response.status_code, 200)
```
Comment (MaximilianWagner) [inline embark/users/test_users.py:32]: You could use http.HTTPStatus.OK for readability from https://docs.python.org/3/library/http.html#module-http.

File: embark/users/test_users.py [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        Signin api expects two keys in the body.
+        email and password. On getting wrong body.
+        it throws 200 status code with content: ""User data is invalid""
+
+        Returns:
+
+        """
+
+        response = self.client.post('/signin', {'username': 'testuser', 'password': '12345'})
+        self.assertEqual(response.status_code, 200)
```
Comment (MaximilianWagner) [inline embark/users/test_users.py:45]: see above.

**Review by MaximilianWagner — APPROVED** [review]
Looks good, except a little annotation about using constants to improve readability.

File: embark/users/test_users.py (hunk 1 above)
Comment (ashutoshsingh0223) [reply embark/users/test_users.py]: Done

File: embark/users/test_users.py (hunk 2 above)
Comment (ashutoshsingh0223) [reply embark/users/test_users.py]: Done

File: embark/users/test_users.py (hunk 3 above)
Comment (ashutoshsingh0223) [reply embark/users/test_users.py]: Done

**Pull request merged**